src/main/haskell/kore/app/exec/Main.hs

```diff
@@ -428,8 +430,8 @@ mainWithOptions
                             (orPredicate, _proof) =
                                 give symbolOrAliasSorts
                                 $ makeMultipleOrPredicate
-                                $ map
-                                    ExpandedPattern.substitutionToPredicate
+                                $ fmap
```

Comment:
Why the map->fmap change?

Reply:
I apologize for the extra noise, but I consider `map` a legacy (pre-`Functor`) and I thought the general consensus is to always use `fmap`. I can revert if people disagree.